src/utils/theme/index.js

```diff
@@ -40,9 +41,11 @@ export const COLORS = {
 
   // new color
   signed_primary: '#4782D7',
+  signed_primary_15: '#4782D726',
   signed_secondary: '#6295DD',
   default_signed_secondary: '#69A2F2',
   anon_primary: '#107793',
+  anon_primary_15: '#10779326',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color codes for `signed_primary_15` and `anon_primary_15` seem to be in the wrong format. The last two characters represent the alpha value in hexadecimal, where 26 hex is approximately 15% opacity. However, it seems like a mistake as 26 hex is actually around 15% of FF (255 in decimal), not 100%. If you want 15% opacity, the correct hex value should be 26 (38 in decimal) which is approximately 15% of 255.

```diff
-   signed_primary_15: '#4782D726',
+   signed_primary_15: '#4782D738',
-   anon_primary_15: '#10779326',
+   anon_primary_15: '#10779338',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->